app/src/main/java/ingsw/group1/findmyphone/managing/MessageBuilder.java

```diff
@@ -0,0 +1,101 @@
+package ingsw.group1.findmyphone.managing;
+
+import android.location.Location;
+
+import androidx.annotation.NonNull;
+
+import com.eis.smslibrary.SMSMessage;
+import com.eis.smslibrary.SMSPeer;
+
+import ingsw.group1.findmyphone.managing.alarm.AlarmManager;
+import ingsw.group1.findmyphone.managing.location.LocationManager;
+
+/**
+ * Builder class useful for creating Request/Response messages
+ *
+ * @author Turcato
+ */
+public class MessageBuilder {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'MessageBuilder' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5068369)

Reply:
> I was thinking about putting the methods of those classes inside the message builder, the position should still be retrieved by the Location Manager, but it's the background service that calls both and answers

Ok, I did not do that at the end, because in my opinion, it's too much code for a single class, but I did reorganize the packages, the result is the following:
![image](https://user-images.githubusercontent.com/56435959/75387825-33056200-58e4-11ea-800e-a8e4e7ac4f85.png)

Everyone, tell me your thoughts pls